server/app/app.py

```diff
@@ -5,27 +5,28 @@
 from flask_compress import Compress
 from flask_cors import CORS
 
-from .rest_api.rest import get_api_resources
-from .util.utils import Float32JSONEncoder
-from .web import webapp
+from server.app.rest_api.rest import get_api_resources
+from server.app.util.utils import Float32JSONEncoder
+from server.app.web import webapp
 
-REACTIVE_LIMIT = 1_000_000
-
-app = Flask(__name__, static_folder="web/static")
-app.json_encoder = Float32JSONEncoder
-cache = Cache(app, config={"CACHE_TYPE": "simple", "CACHE_DEFAULT_TIMEOUT": 860_000})
-Compress(app)
-CORS(app)
+# Application Data
```

Comment:
In some scenarios, I can see us wanting to load and serve multiple data sets in one process (eg, using the future API in a notebook).  Have you considered making the "app" a class, and store data and cache within the instance?

Reply:
👍 Done